fab/fabfile.py

```diff
@@ -510,6 +510,7 @@ def _deploy_without_asking():
         execute_with_timing(db.ensure_preindex_completion)
 
         # handle static files
+        execute_with_timing(staticfiles.prime_version_static)
```

Comment:
You really want to put this before `ensure_preindex_completion` if you want to keep the pipelining aspect of it.